### ISSUE TITLE
README for check_maps is out of date

### ISSUE THREAD

**Issue by gold2718** [ESMCI/cime#1946]
The README for `check_maps` (<root>/tools/mapping/check_maps/README) has instructions for Yellowstone that do not work (can't check Cheyenne today). In particular:
```
module load esmf

Lmod Warning: Did not find: esmf
```

I'm not sure what the correct action is.

**Comment by mvertens** [comment]
@fischer - could you please put in an issue and assign it to yourself. We
definitely need to fix this before the release.

**Comment by gold2718** [comment]
Using the Yellowstone entry in config_machines.xml as a guide, I found a sequence that works:
```
module purge
module load ncarenv/1.0
module load ncarbinlibs/1.1
module load gmake/4.1
module load intel/15.0.3
module load esmf
module load esmf-7.0.0-ncdfio-uni-O
```
There is probably a smaller set but the load of `intel/15.0.3` seems required.

**Comment by jedwards4b** [comment]
If you run configure for a case it will create an .env_mach_specific.sh (or .csh) which you can then source to get this result.

**Comment by gold2718** [comment]
That is useful but would be an awkward way to build these offline tools. Is there (or could there be) a top-level command that just generates env_mach_specific.(c)sh using all the CIME machinery around config_machines.xml? That would really streamline keeping our various toolchains up to date.

**Comment by ekluzek** [comment]
Could this be done for CESM2.0?

**Comment by fischer-ncar** [comment]
Fixed via #2589
